env/custom.sh

```diff
@@ -1,3 +1,3 @@
 # set e.g.
 # export CBIOPORTAL_URL="http://localhost:8080"
-export BACKEND=dippindots:override_option_when_import
+export BACKEND=cBioPortal:release-3.2.0
```

Comment:
@dippindots i think we can remove this line, the backend image is determined based on the branch it is running on

Reply:
see e.g. https://github.com/cBioPortal/cbioportal-frontend/blob/master/env/custom.sh

Reply:
Thanks @inodb, sure I can remove this line. Since we decide to postpone the change to the path of the endpoint, should I just change this pr to `update custom.sh`?